dts/riscv/wch/ch32v208/ch32v208.dtsi

```diff
@@ -159,5 +159,5 @@
 };
 
 &cpu0 {
-	clock-frequency = <DT_FREQ_M(144)>;
+	clock-frequency = <DT_FREQ_M(120)>;
```

Comment:
There seems to be a bunch of unrelated changes in this commit?

Reply:
It is not unrelated, it is recommended to not run CPU over 120MHz when using the whole flash.

Reply:
![image](https://github.com/user-attachments/assets/17abf1eb-b848-4617-935f-d6ab16d99ab8)

Reply:
(it's also the frequency for using USB, so it's better to have it be the default)

Reply:
Moreover, 
![image](https://github.com/user-attachments/assets/31390ac3-c3a4-4235-8455-a5b56e4dffa2)
It's also something to think about for USB.

Reply:
oh wow okay -- maybe to avoid a similar double-take from others in the future, it might be worth adding just a bit more context to the commit message then ("... and apply datasheet recommendations w.r.t CPU clock speed etc")?

Reply:
BTW it's actually running in 240M -> prescaler /2 mode atm, from HSE 16 * 15 -> 240 /2 -> 120, so in theory this is the way it should stay for the future (when using HSE, with HSI i dont think it can get to 240M at all so it would run at 60Mhz if the prescaler isnt changed?)
Added the change description to the commit.